-------------------------------------------------------------------------------
Test set: newser.AcceptanceTestSuite
-------------------------------------------------------------------------------
Tests run: 46, Failures: 3, Errors: 1, Skipped: 0, Time elapsed: 459.979 sec <<< FAILURE! - in newser.AcceptanceTestSuite
Then form must not be submitted(newser.steps.serenity.ChaticonDefinition)  Time elapsed: 20.884 sec  <<< FAILURE!
java.lang.AssertionError
	at newser.steps.serenity.ChaticonDefinition.Thenformmustnotbesubmitted(ChaticonDefinition.java:159)

Scenario: Chat form validation  Time elapsed: 21.695 sec  <<< FAILURE!
java.lang.AssertionError: scenario failed!

Chat.story  Time elapsed: 21.718 sec  <<< FAILURE!
java.lang.AssertionError: story failed!

newser.AcceptanceTestSuite  Time elapsed: 7.606 sec  <<< ERROR!
java.lang.RuntimeException: 
org.jbehave.core.embedder.Embedder$RunningStoriesFailed: Failures in running stories: 
org.jbehave.core.embedder.PerformableTree$SomethingHappened@660f0be7: java.lang.AssertionError
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
Caused by: org.jbehave.core.embedder.Embedder$RunningStoriesFailed: 
Failures in running stories: 
org.jbehave.core.embedder.PerformableTree$SomethingHappened@660f0be7: java.lang.AssertionError
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)

